modules/javafx.controls/src/main/java/javafx/scene/control/ScrollPane.java

```diff
@@ -73,8 +73,10 @@
  * node in a Group.
  * <p>
  * ScrollPane sets focusTraversable to false.
- * </p>
- *
+ * <p>
+ * ScrollPane only acts on key presses when it has the focus ({@link #isFocused()}
+ * returns {@code true}) and won't respond to unconsumed key events that bubble
```

Comment:
what about consumed events?  does it respond to those?

i.e. you may want to remove the word 'unconsumed'